js/translator.js

```diff
@@ -0,0 +1,9 @@
+let languages = {
```

Comment:
With this function I want my output to change on it's own depending on my my language variable.